wfl/Database/MongoDBManager.cs

```diff
@@ -63,9 +64,21 @@ private string GenerateCollectionName<T>()
         public async Task<T> GetEntityByIdAsync<T>(string id) where T: IEntity
         {
             IMongoCollection<T> collection = db.GetCollection<T>(GenerateCollectionName<T>());
-            IAsyncCursor<T> task = await collection.FindAsync(e => e.Id == id, null);
+            var filter = Builders<T>.Filter.Eq("Id", id);
+            IAsyncCursor<T> task = await collection.FindAsync(filter, null);
             return await task.FirstOrDefaultAsync();
         }
+        public async Task<IEnumerable<T>> GetEntitiesByDateAsync<T>(DateTime date) where T : IEntity
```

Comment:
Is this retrieving entities by create date or something? What date is being used as comparison here?